aea/configurations/base.py

```diff
@@ -1878,7 +1878,9 @@ def __init__(
 
 def _check_aea_version(package_configuration: PackageConfiguration) -> None:
     """Check the package configuration version against the version of the framework."""
-    current_aea_version = Version(__aea_version__)
+    # temp ignore pre-release!
+    version = __aea_version__[:-5] if "rc" in __aea_version__ else __aea_version__
```

Comment:
`1.0.0-rc.1<1.0.0`. But it makes no sense to go through the pain of having the package bounds updated in two stages. So for the release candidates we use this hack.

Reply:
You may want to use `__aea_version__[:5]` (just in case we go up to rc with two digits :-) )